src/v2/view-builder/internals/FormInputFactory.js

```diff
@@ -144,6 +147,29 @@ const createIdpButtons = (remediations) => {
     };
     return button;
   });
+
+  return [...pivButton, ...idpButtons];
```

Comment:
Double checking on the ordering - do we want PIV to take priority?

Reply:
yup piv should always take priority when available. Checked that with PM